frame/contracts/proc-macro/src/lib.rs

```diff
@@ -316,8 +326,20 @@ impl EnvDef {
 			.ok_or(err("Invalid environment definition, expected `mod` to be inlined."))?
 			.1;
 
+		let aliases = items.iter().filter(|i| {
+			let i = match i {
+				syn::Item::Fn(i_fn) => i_fn,
+				_ => return false,
+			};
+
+			i.attrs.iter().any(|a| {
+				a.path.get_ident().map(|i| i.to_string().eq("prefixed_alias")).unwrap_or(false)
+			})
+		});
```

Comment:
Doesn't this just adds the prefixed version twice but adding no non-prefixed version?

Reply:
The duplicates are then [renamed](https://github.com/paritytech/substrate/pull/12126/files/2c8a07bac1988773a10174440eaa6ec673aba3ee#diff-ccbc33017b6a98097a085fc49556ae57f92f446fd44953b7cdcaafc67269c7a7R192) during each function code expansion.

Reply:
This is my point. They are renamed in **both** instances to have the prefix. So you have no unprefixed version. I just did a quick check and removed the `seal_` prefix from `seal_now` which makes the test fail.

Reply:
You're right, this is a sad and stupid mistake.